### PR TITLE
Add unschedualbe to infra

### PR DESCRIPTION
**Pull request by glennswest**
1. Add a ansible group for infra nodes
2. Add unschedulable tag infra nodes
3. Remove swap off due to waalinux issue for now.

### CODE REVIEW

File: reference-architecture/azure-ansible/bastion.sh [hunk 1]
```diff
@@ -325,6 +325,11 @@ cat > /home/${AUSERNAME}/setup-azure-node.yml <<EOF
   - name: make sure its set to unsceduable
     command: oadm manage-node {{inventory_hostname}} --schedulable=false
     delegate_to: master1
+- hosts: infranodes
+  tasks:
+  - name: make sure its set to unsceduable
+    command: oadm manage-node {{inventory_hostname}} --schedulable=false
```
Comment (e-minguez) [inline reference-architecture/azure-ansible/bastion.sh]: If the infranodes are not schedulable they won't be able to host the registry/router/etc.

File: reference-architecture/azure-ansible/node.sh [hunk 2]
```diff
@@ -26,9 +26,4 @@ DATA_SIZE=95%VG
 EXTRA_DOCKER_STORAGE_OPTIONS="--storage-opt dm.basesize=3G"
 EOF
 
-# Disable swap
-sed -i -e 's/ResourceDisk.EnableSwap=y/ResourceDisk.EnableSwap=n/g' /etc/waagent.conf
-swapoff -a
```
Comment (e-minguez) [inline reference-architecture/azure-ansible/node.sh]: I think we need to fix it instead removing it

**Comment by glennswest** [comment]
Did u do the pr?

**Comment by glennswest** [comment]
Retest

**Comment by glennswest** [comment]
I'm testing it 
Let u know

**Pull request closed without merging**